diff --git a/node_modules/react-native-webview/android/.classpath b/node_modules/react-native-webview/android/.classpath
new file mode 100644
index 0000000..bbe97e5
--- /dev/null
+++ b/node_modules/react-native-webview/android/.classpath
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-17/"/>
+	<classpathentry kind="con" path="org.eclipse.buildship.core.gradleclasspathcontainer"/>
+	<classpathentry kind="output" path="bin/default"/>
+</classpath>
diff --git a/node_modules/react-native-webview/android/.gradle/8.9/checksums/checksums.lock b/node_modules/react-native-webview/android/.gradle/8.9/checksums/checksums.lock
new file mode 100644
index 0000000..df6c62b
Binary files /dev/null and b/node_modules/react-native-webview/android/.gradle/8.9/checksums/checksums.lock differ
diff --git a/node_modules/react-native-webview/android/.gradle/8.9/dependencies-accessors/gc.properties b/node_modules/react-native-webview/android/.gradle/8.9/dependencies-accessors/gc.properties
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-webview/android/.gradle/8.9/executionHistory/executionHistory.lock b/node_modules/react-native-webview/android/.gradle/8.9/executionHistory/executionHistory.lock
new file mode 100644
index 0000000..1974e53
Binary files /dev/null and b/node_modules/react-native-webview/android/.gradle/8.9/executionHistory/executionHistory.lock differ
diff --git a/node_modules/react-native-webview/android/.gradle/8.9/fileChanges/last-build.bin b/node_modules/react-native-webview/android/.gradle/8.9/fileChanges/last-build.bin
new file mode 100644
index 0000000..f76dd23
Binary files /dev/null and b/node_modules/react-native-webview/android/.gradle/8.9/fileChanges/last-build.bin differ
diff --git a/node_modules/react-native-webview/android/.gradle/8.9/fileHashes/fileHashes.lock b/node_modules/react-native-webview/android/.gradle/8.9/fileHashes/fileHashes.lock
new file mode 100644
index 0000000..65d11bb
Binary files /dev/null and b/node_modules/react-native-webview/android/.gradle/8.9/fileHashes/fileHashes.lock differ
diff --git a/node_modules/react-native-webview/android/.gradle/8.9/gc.properties b/node_modules/react-native-webview/android/.gradle/8.9/gc.properties
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-webview/android/.gradle/9.2.0/checksums/checksums.lock b/node_modules/react-native-webview/android/.gradle/9.2.0/checksums/checksums.lock
new file mode 100644
index 0000000..76f527c
Binary files /dev/null and b/node_modules/react-native-webview/android/.gradle/9.2.0/checksums/checksums.lock differ
diff --git a/node_modules/react-native-webview/android/.gradle/9.2.0/checksums/md5-checksums.bin b/node_modules/react-native-webview/android/.gradle/9.2.0/checksums/md5-checksums.bin
new file mode 100644
index 0000000..fd7745d
Binary files /dev/null and b/node_modules/react-native-webview/android/.gradle/9.2.0/checksums/md5-checksums.bin differ
diff --git a/node_modules/react-native-webview/android/.gradle/9.2.0/checksums/sha1-checksums.bin b/node_modules/react-native-webview/android/.gradle/9.2.0/checksums/sha1-checksums.bin
new file mode 100644
index 0000000..27d5a40
Binary files /dev/null and b/node_modules/react-native-webview/android/.gradle/9.2.0/checksums/sha1-checksums.bin differ
diff --git a/node_modules/react-native-webview/android/.gradle/9.2.0/fileChanges/last-build.bin b/node_modules/react-native-webview/android/.gradle/9.2.0/fileChanges/last-build.bin
new file mode 100644
index 0000000..f76dd23
Binary files /dev/null and b/node_modules/react-native-webview/android/.gradle/9.2.0/fileChanges/last-build.bin differ
diff --git a/node_modules/react-native-webview/android/.gradle/9.2.0/fileHashes/fileHashes.bin b/node_modules/react-native-webview/android/.gradle/9.2.0/fileHashes/fileHashes.bin
new file mode 100644
index 0000000..63b9feb
Binary files /dev/null and b/node_modules/react-native-webview/android/.gradle/9.2.0/fileHashes/fileHashes.bin differ
diff --git a/node_modules/react-native-webview/android/.gradle/9.2.0/fileHashes/fileHashes.lock b/node_modules/react-native-webview/android/.gradle/9.2.0/fileHashes/fileHashes.lock
new file mode 100644
index 0000000..eaa66b2
Binary files /dev/null and b/node_modules/react-native-webview/android/.gradle/9.2.0/fileHashes/fileHashes.lock differ
diff --git a/node_modules/react-native-webview/android/.gradle/9.2.0/gc.properties b/node_modules/react-native-webview/android/.gradle/9.2.0/gc.properties
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-webview/android/.gradle/buildOutputCleanup/buildOutputCleanup.lock b/node_modules/react-native-webview/android/.gradle/buildOutputCleanup/buildOutputCleanup.lock
new file mode 100644
index 0000000..b1a3e13
Binary files /dev/null and b/node_modules/react-native-webview/android/.gradle/buildOutputCleanup/buildOutputCleanup.lock differ
diff --git a/node_modules/react-native-webview/android/.gradle/buildOutputCleanup/cache.properties b/node_modules/react-native-webview/android/.gradle/buildOutputCleanup/cache.properties
new file mode 100644
index 0000000..edfc554
--- /dev/null
+++ b/node_modules/react-native-webview/android/.gradle/buildOutputCleanup/cache.properties
@@ -0,0 +1,2 @@
+#Fri Sep 04 19:29:03 ICT 2026
+gradle.version=8.9
diff --git a/node_modules/react-native-webview/android/.gradle/vcs-1/gc.properties b/node_modules/react-native-webview/android/.gradle/vcs-1/gc.properties
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-webview/android/.project b/node_modules/react-native-webview/android/.project
new file mode 100644
index 0000000..ec38997
--- /dev/null
+++ b/node_modules/react-native-webview/android/.project
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>react-native-webview</name>
+	<comment>Project react-native-webview created by Buildship.</comment>
+	<projects>
+	</projects>
+	<buildSpec>
+		<buildCommand>
+			<name>org.eclipse.jdt.core.javabuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+		<buildCommand>
+			<name>org.eclipse.buildship.core.gradleprojectbuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+	</buildSpec>
+	<natures>
+		<nature>org.eclipse.jdt.core.javanature</nature>
+		<nature>org.eclipse.buildship.core.gradleprojectnature</nature>
+	</natures>
+	<filteredResources>
+		<filter>
+			<id>1788421469702</id>
+			<name></name>
+			<type>30</type>
+			<matcher>
+				<id>org.eclipse.core.resources.regexFilterMatcher</id>
+				<arguments>node_modules|\.git|__CREATED_BY_JAVA_LANGUAGE_SERVER__</arguments>
+			</matcher>
+		</filter>
+	</filteredResources>
+</projectDescription>
diff --git a/node_modules/react-native-webview/android/.settings/org.eclipse.buildship.core.prefs b/node_modules/react-native-webview/android/.settings/org.eclipse.buildship.core.prefs
new file mode 100644
index 0000000..1675490
--- /dev/null
+++ b/node_modules/react-native-webview/android/.settings/org.eclipse.buildship.core.prefs
@@ -0,0 +1,2 @@
+connection.project.dir=../../../android
+eclipse.preferences.version=1
diff --git a/node_modules/react-native-webview/android/.settings/org.eclipse.jdt.core.prefs b/node_modules/react-native-webview/android/.settings/org.eclipse.jdt.core.prefs
new file mode 100644
index 0000000..626e0e1
--- /dev/null
+++ b/node_modules/react-native-webview/android/.settings/org.eclipse.jdt.core.prefs
@@ -0,0 +1,4 @@
+eclipse.preferences.version=1
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=17
+org.eclipse.jdt.core.compiler.compliance=17
+org.eclipse.jdt.core.compiler.source=17
diff --git a/node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewClient.java b/node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewClient.java
index 251939e..09eb84a 100644
--- a/node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewClient.java
+++ b/node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewClient.java
@@ -161,6 +161,10 @@ public class RNCWebViewClient extends WebViewClient {
 
     @Override
     public void onReceivedSslError(final WebView webView, final SslErrorHandler handler, final SslError error) {
+        // [FIX] Bỏ qua lỗi chứng chỉ SSL cho môi trường dev/sandbox của VNPAY
+        handler.proceed();
+        if (true) return;
+
         // onReceivedSslError is called for most requests, per Android docs: https://developer.android.com/reference/android/webkit/WebViewClient#onReceivedSslError(android.webkit.WebView,%2520android.webkit.SslErrorHandler,%2520android.net.http.SslError)
         // WebView.getUrl() will return the top-level window URL.
         // If a top-level navigation triggers this error handler, the top-level URL will be the failing URL (not the URL of the currently-rendered page).
